src/components/atoms/bar-chart-content.tsx

```diff
@@ -102,6 +104,28 @@ export const BarChartContent: FC<BarChartContentProps> = ({
     <TrendLine lineWidth={3} color={primaryColor} {...props} />
   );
 
+  const Label: FC<{
+    data: any;
+    y: (value: number) => void;
+    x: () => void;
+    bandwidth: number;
+  }> = (props) => {
```

Comment:
This can be `FC<BarChildProps>`, we've already defined the types. That'll also fix the type warning on `y(value.value)` below which returns `number` not `void`